src/main/telemetry/jetiexbus.c

```diff
@@ -552,7 +619,7 @@ uint8_t sendJetiExBusTelemetry(uint8_t packetID, uint8_t item)
     }
 
     serialWriteBuf(jetiExBusPort, jetiExBusTelemetryFrame, jetiExBusTelemetryFrame[EXBUS_HEADER_MSG_LEN]);
-    jetiExBusTransceiveState = EXBUS_TRANS_IS_TX_COMPLETED;
+    jetiExBusDisableTx();
```

Comment:
(maybe: `jetiExBusTxDone` is more accurate name)

Reply:
Agree - updated